static/src/stylesheets/layout/_side-margins.scss

```diff
@@ -30,7 +30,7 @@
             content: '';
             position: absolute;
             z-index: 1;
-            background: rgba($brightness-7,  .05);
+            background: rgba(0, 0, 0,  .03);
```

Comment:
![red_excited tears](https://user-images.githubusercontent.com/11539094/45498515-128a2c00-b772-11e8-98f3-4fc8dcd4f36b.png)

Reply:
What?

Reply:
i'm happy zef this is me being happy

Reply:
It's so scary!